internal/resource/configmap_test.go

```diff
@@ -251,7 +251,12 @@ listeners.ssl.default = 5671
 management.ssl.certfile   = /etc/rabbitmq-tls/tls.crt
 management.ssl.keyfile    = /etc/rabbitmq-tls/tls.key
 management.ssl.port       = 15671
-management.tcp.port       = 15672
+
+prometheus.ssl.certfile = /etc/rabbitmq-tls/tls.crt
+prometheus.ssl.keyfile  = /etc/rabbitmq-tls/tls.key
+prometheus.ssl.port     = 15691
+management.tcp.port     = 15672
```

Comment:
What do you think, would this read better with an extra newline?

Reply:
done via https://github.com/rabbitmq/cluster-operator/pull/533/commits/97ce18d95bf0edab8cba95b565dccc0819636763